<ui version="4.0" >
 <class>AddEditMovieDlg</class>
 <widget class="QDialog" name="AddEditMovieDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>My Movies - Add Movie</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="5" column="4" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2" >
    <widget class="QLineEdit" name="locationLineEdit" />
   </item>
   <item row="1" column="4" >
    <widget class="QDateEdit" name="acquiredDateEdit" >
     <property name="alignment" >
      <set>Qt::AlignRight</set>
     </property>
     <property name="displayFormat" >
      <string>ddd MMM d, yyyy</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;Title:</string>
     </property>
     <property name="buddy" >
      <cstring>titleLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5" >
    <widget class="QLineEdit" name="titleLineEdit" />
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>&amp;Notes:</string>
     </property>
     <property name="buddy" >
      <cstring>notesTextEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6" >
    <widget class="QTextEdit" name="notesTextEdit" >
     <property name="tabChangesFocus" >
      <bool>true</bool>
     </property>
     <property name="lineWrapMode" >
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>&amp;Minutes:</string>
     </property>
     <property name="buddy" >
      <cstring>minutesSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QSpinBox" name="minutesSpinBox" >
     <property name="alignment" >
      <set>Qt::AlignRight</set>
     </property>
     <property name="specialValueText" >
      <string>Unknown</string>
     </property>
     <property name="maximum" >
      <number>720</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>&amp;Location:</string>
     </property>
     <property name="buddy" >
      <cstring>locationLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>A&amp;cquired:</string>
     </property>
     <property name="buddy" >
      <cstring>acquiredDateEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QSpinBox" name="yearSpinBox" >
     <property name="alignment" >
      <set>Qt::AlignRight</set>
     </property>
     <property name="specialValueText" >
      <string>Unknown</string>
     </property>
     <property name="maximum" >
      <number>2100</number>
     </property>
     <property name="minimum" >
      <number>1890</number>
     </property>
     <property name="value" >
      <number>1890</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;Year:</string>
     </property>
     <property name="buddy" >
      <cstring>yearSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleLineEdit</tabstop>
  <tabstop>yearSpinBox</tabstop>
  <tabstop>minutesSpinBox</tabstop>
  <tabstop>acquiredDateEdit</tabstop>
  <tabstop>locationLineEdit</tabstop>
  <tabstop>notesTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEditMovieDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>342</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel" >
     <x>367</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEditMovieDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>459</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel" >
     <x>480</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
